virtual_ecosystem/models/animals/scaling_functions.py

```diff
@@ -264,3 +266,188 @@ def natural_mortality_scaling(mass: float, terms: tuple) -> float:
     daily_mortality_prob = 1 - exp(-daily_mortality_rate)
 
     return daily_mortality_prob
+
+
+def alpha_i_k(alpha_0_herb: float, mass: float) -> float:
+    """Effective rate at which an individual herbivore searches its environment.
+
+    This is linear scaling of herbivore search times with current body mass.
+
+    Madingley
```

Comment:
I get why you're using these function names, but it is harder to follow:
```suggestion
def herbivore_search_rate(alpha_0_herb: float, mass: float) -> float:
    """Effective rate at which an individual herbivore searches its environment.

    This is a linear scaling of herbivore search times with current body mass. It is the :math:`\alpha_{i,k}` term from the Madingley model.

```

Reply:
I should have mentioned it in my original post but these technical method names are temporary until we get into unit-testing but I could probably shorten that timing to the other side of a proper flow diagram. It is... challenging to do the cross-referencing with madingley to make sure it's all assembled properly. If I started changing the names right now it will be extremely difficult.